test/scrolling.html

```diff
@@ -45,6 +45,26 @@
         });
       });
 
+      describe('the height of scroller', () => {
+        beforeEach(() => {
```

Comment:
This test also passes on master. In order to make a proper test you need to tweak `body` height.

Add this line to `beforeEach` to reproduce the case that causes the original issue:
```js
document.body.style.height = '100px';
```

Then also remember to unset it back so that other tests won't be affected:
``` js
afterEach(() => {
  document.body.style.height = null;
});
```

Reply:
done